packages/form-upload/Upload.js

```diff
@@ -182,6 +183,7 @@ const Upload = ({
           allowedFileNameCharacters,
         });
         upload.id = `${upload.id}-${uuid()}`;
+        if (onFilePreUpload) onFilePreUpload(upload, file);
```

Comment:
This should be passed through the `UploadCore` constructor, `onPreStart: onFilePreUpload`
https://github.com/Availity/sdk-js/blob/9b5e81bef0216d13fab6ca0bbe87370e2019f490/packages/upload-core/src/upload.js#L79